ember-cli-build.js

```diff
@@ -25,6 +26,9 @@ module.exports = function(defaults) {
           tree: 'sandbox'
         }
       }
+    },
+    'ember-cli-mirage': {
+      directory: path.resolve(__dirname, path.join('tests', 'dummy', 'mirage'))
```

Comment:
What is this doing?

Reply:
making mirage work when in-repo addons are present :( though I don't understand why it wasn't working, I found this answer in the issues. Let me poke around some more

Reply:
Doesn't seem like a huge deal, I was just curious 🙂